storage/src/main/java/tech/pegasys/artemis/storage/ChainStorageClient.java

```diff
@@ -103,7 +105,8 @@ public void initializeFromGenesis(final BeaconState genesisState) {
           "Failed to set genesis state: store has already been initialized");
     }
 
-    eventBus.post(new StoreGenesisDiskUpdateEvent(store));
+    diskUpdateChannel.onDiskGenesisUpdate(new DiskGenesisUpdate(store));
```

Comment:
We don't need the wrapper `DiskGenesisUpdate` event here - just pass `store` as the argument.

Reply:
Done.